src/topicButton.js

```diff
@@ -1,9 +1,22 @@
 /* exported TopicButton */
-
-const {GObject, Gtk} = imports.gi;
+const {GObject, Gtk, GLib, Gio} = imports.gi;
 
 const {Lockscreen} = imports.lockscreen;
 
+const TopicIface = `
+<node>
+  <interface name="com.endlessm.HackToolbox.topic">
```

Comment:
This is bikeshedding, but I noticed in d-feet that most other DBus interface names were capitalized as in `com.example.AppName.SomeObjectName`, so I'd suggest capitalizing `Topic` for consistency.